### ISSUE TITLE
Failed to execute goal on project creator: Could not resolve dependencies for project com.terge:creator:jar:0.0.1-SNAPSHOT: Could not find artifact com.sun:tools:jar:1.7.0 at specified path /Library/Java/JavaVirtualMachines/jdk-11.0.9.jdk/Contents/Home/../lib/tools.jar -> [Help 1]

### ISSUE THREAD

**Issue by Drjacky** [Terge3141/WhatsIllustrator#22]
```scala
[INFO] PDF-Converter for Whatsapp Messages ................ SUCCESS [  0.002 s]
[INFO] PDF-Converter for Whatsapp Messages Helper ......... SUCCESS [  7.031 s]
[INFO] PDF-Converter for Whatsapp Messages Configurator ... SUCCESS [  0.302 s]
[INFO] PDF-Converter for Whatsapp Messages ImageMatcher ... SUCCESS [  0.244 s]
[INFO] PDF-Converter for Whatsapp Messages Message Parser . SUCCESS [  6.093 s]
[INFO] PDF-Converter for Whatsapp Messages Creator ........ FAILURE [  4.192 s]
[INFO] PDF-Converter for Whatsapp Messages Videothumbnails  SKIPPED
[INFO] PDF-Converter for Whatsapp Messages FopCreator ..... SKIPPED
[INFO] PDF-Converter for Whatsapp Messages TexCreator ..... SKIPPED
[INFO] PDF-Converter for Whatsapp Messages OdfCreator ..... SKIPPED
[INFO] PDF-Converter for Whatsapp Messages Program ........ SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  17.987 s
[INFO] Finished at: 2022-10-23T10:56:58+01:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal on project creator: Could not resolve dependencies for project com.terge:creator:jar:0.0.1-SNAPSHOT: Could not find artifact com.sun:tools:jar:1.7.0 at specified path /Library/Java/JavaVirtualMachines/jdk-11.0.9.jdk/Contents/Home/../lib/tools.jar -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException
[ERROR]
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <args> -rf :creator
```

But I have java 11 installed and set properly:
```bash
java --version
java 11.0.9 2020-10-20 LTS
######
mvn -version
Apache Maven 3.8.6 (84538c9988a25aec085021c365c560670ad80f63)
Maven home: /usr/local/Cellar/maven/3.8.6/libexec
Java version: 11.0.9, vendor: Oracle Corporation, runtime: /Library/Java/JavaVirtualMachines/jdk-11.0.9.jdk/Contents/Home
```

**Comment by Drjacky** [comment]
I found this:
![image](https://user-images.githubusercontent.com/2296754/197386048-b824589c-fcef-494c-8927-e9ae82fef8ae.png)
https://docs.oracle.com/en/java/javase/13/migrate/migration-guide.pdf

Seems tools.jar was removed from jdk **since** version 9

**Comment by Terge3141** [comment]
It works with openjdk:

java -version
openjdk version "11.0.17" 2022-10-18
OpenJDK Runtime Environment (build 11.0.17+8-post-Ubuntu-1ubuntu222.04)
OpenJDK 64-Bit Server VM (build 11.0.17+8-post-Ubuntu-1ubuntu222.04, mixed mode, sharing)

Going to check it with oracle

**Comment by Terge3141** [comment]
Fixed with commit 41c74e42fc9e37c3fd305ba75d7416e8ab6daccd
